src/library/tableitemdelegate.h

```diff
@@ -26,3 +26,8 @@ class TableItemDelegate : public QStyledItemDelegate {
   private:
     QTableView* m_pTableView;
 };
+
+void paintItemBackground(
```

Comment:
This could become a protected, static function in `TableItemDelegate`

Reply:
Done.